components/cosmetic_filters/content/renderer/BUILD.gn

```diff
@@ -0,0 +1,28 @@
+import("//build/config/features.gni")
```

Comment:
the path for this should just be `components/cosmetic_filters/renderer` see https://www.chromium.org/developers/design-documents/cookbook

Reply:
corrected